### PR TITLE
Feature: PPP Multiple Option Conversion

### PR DESCRIPTION
**Pull request by joeflack4**
Added the ability to output multiple files as a product of 'n' supplied options.

*Example*
```
$ ls
$ # Shows 6 XlsForm excel files.
$ python3 -m pmix.ppp /* -f html doc -p full minimal -l English Yoruba
$ ls
$ # Shows 6 XlsForm excel files and 48 output files as a result of languages and options chosen, i.e. 6 input files * 2 file formats * 2 preset option formats * 2 languages.
```

### CODE REVIEW

File: test/test_ppp.py [hunk 1]
```diff
@@ -283,6 +283,45 @@ def test_multi_conversion(self):
...
+        src_dir_ls_input = MultiConversionTest.ls(src_dir)
+        src_files = \
+            [src_dir + x for x in src_dir_ls_input if x.endswith('.xlsx')]
+        subprocess.call(['rm', '-rf', out_dir])
+        subprocess.call(['mkdir', out_dir])
+        subprocess.call(['python', '-m', 'pmix.ppp'] + src_files +
+                        ['-o', out_dir, '-f', 'doc', '-p', 'minimal',
+                         '-l', 'English', 'Français'])
+        out_dir_ls_input = MultiConversionTest.ls(out_dir)
+        expected_output = "['/Users/joeflack4/projects/pmix_dev/test/files/mu" \
```
Comment (jkpr) [inline test/test_ppp.py]: Why are you adding an absolute path here? It won't work on other computers.

File: test/test_ppp.py [hunk 2]
```diff
@@ -283,6 +283,45 @@ def test_multi_conversion(self):
         self.assertEqual(str(out_dir_ls_input), expected_output)
 
 
+class MultiConversionTest(unittest.TestCase):
+    """Test conversion of n files in n languages for n option combinations."""
+
+    @staticmethod
+    def ls(_dir):
```
Comment (jkpr) [inline test/test_ppp.py]: Why this and not python's `os.listdir()` ?

File: pmix/ppp/__init__.py [hunk 3]
```diff
@@ -84,6 +89,45 @@ def convert_file(in_file, language=None, outpath=None, **kwargs):
         raise OdkException(err)
 
 
+def enumerate_combos(dict_with_lists):
```
Comment (jkpr) [inline pmix/ppp/__init__.py]: This function seems overly complicated. Try instead:

`[dict(zip(dict_with_lists, item)) for item in itertools.product(*dict_with_lists.values())]`

The order is guaranteed to be the same. See https://docs.python.org/3/library/stdtypes.html#dictionary-view-objects

File: test/test_ppp.py (hunk 1 above)
Comment (joeflack4) [reply test/test_ppp.py]: Forgot to update. Fixed!

File: pmix/ppp/__init__.py (hunk 3 above)
Comment (joeflack4) [reply pmix/ppp/__init__.py]: Over complicated indeed. There is a pythonic way for that (it seems)! Updated.

File: test/test_ppp.py (hunk 2 above)
Comment (joeflack4) [reply test/test_ppp.py]: Looks like there is a pythonic way for that! Updated.

**Comment by joeflack4** [comment]
@jkpr All issues have been addressed!

**Pull request merged**